waiter/src/waiter/state.clj

```diff
@@ -587,9 +585,12 @@
   "Launches a go-block that sends a message to unblacklist-instance-chan to unblacklist instance-id after blacklist-period-ms have elapsed."
   [correlation-id instance-id blacklist-period-ms unblacklist-instance-chan]
   (async/go
-    (async/<! (async/timeout blacklist-period-ms))
-    (cid/cinfo correlation-id "requesting instance" instance-id "to be unblacklisted")
-    (async/>! unblacklist-instance-chan {:instance-id instance-id})))
+    (try
+      (async/<! (async/timeout blacklist-period-ms))
+      (cid/cinfo correlation-id "requesting instance" instance-id "to be unblacklisted")
+      (async/>! unblacklist-instance-chan {:instance-id instance-id})
+      (catch Throwable th
+        (log/error th "unexpected error inside trigger-unblacklist-process")))))
```

Comment:
Can we use `cid/cerror` here? Also, can we include the `instance-id` and `blacklist-period-ms` in the log message?